controller/jobspecController.ts

```diff
@@ -0,0 +1,20 @@
+import { JobSpec } from '../model/jobSpec';
+import { Application, Request, Response } from "express";
+const jobService = require('../service/jobspecService')
+
+module.exports = function(app: Application){
+
+    app.get('/job-spec/:id', async (req: Request, res: Response) =>{
+        let data:JobSpec;
+
+        try {
+            data = await jobService.getJobspecById(req.params.id)
+        } catch (e) {
+            console.error(e);
```

Comment:
Would it make sense here that if you can't find a job spec you should show a `404` page to the user?